doc/admin/conf_files/krb5_conf.rst

```diff
@@ -212,6 +212,12 @@ The libdefaults section may contain any of the following relations:
     data), and anything the fake KDC sends will not be trusted without
     verification using some secret that it won't know.
 
+**dns_uri_lookup**
```

Comment:
This text should indicate the default value, and should say "New in release 1.15." at the end.